compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/MetalRedrawer.uikit.kt

```diff
@@ -461,6 +464,46 @@ internal class MetalRedrawer(
     companion object {
         private val renderingDispatchQueue =
             dispatch_queue_create("RenderingDispatchQueue", null)
+
+        private class CachedCommandQueue(
```

Comment:
I'd move it out from `companion object` and expose methods instead of public properties